src/models/children.js

```diff
@@ -1,42 +1,62 @@
 import uuid from 'uuid/v4';
 import knex, { likeFilter, exactFilter } from '../utils/db';
 
-export const dbGetChildren = filters =>
-  knex('children')
+export const dbGetChildren = filters => {
+  let threeMonthsAgo = new Date();
+  threeMonthsAgo.setMonth(threeMonthsAgo.getMonth() - 3);
+  threeMonthsAgo = Math.round(threeMonthsAgo.getTime() / 1000);
+
+  let query = knex('children')
     .select([
       'children.*',
       'employees.name as assigneeName',
-      'prevFeedback.prevFeedbackDate',
+      'feedback.created as lastFeedbackDate',
+      knex.raw(`case when feedback.created < to_timestamp(${threeMonthsAgo}) then 1 else 0 end as alert`),
```

Comment:
Something like `knex.raw('case when feedback.created < to_timestamp(?) then 1 else 0 end as alert', threeMonthsAgo)` should be safer